test/e2e-oidc/external_oidc.go

```
@@ -0,0 +1,1239 @@
+
+	testClient.validateOAuthState(t, testCtx, true, newExternalOIDCArchitectureEnabled)
+
+	testClient.testOIDCAuthentication(t, testCtx, kcClient, "", "", false)
+	t.Logf("<<< Sub-test 6 PASSED: authentication failed as expected with unknown claim")
+
+	t.Logf("=== All OIDC sub-tests PASSED ===")
+}
+
+// All helper functions from original file
+
+type oidcAuthResponse struct {
```

golangci-lint (2.12.2)

[error] type oidcAuthResponse is unused

(unused)